ajax/libs/jqwidgets/package.json

```diff
@@ -0,0 +1,140 @@
+{
+  "name": "jqwidgets",
+  "description": "jQWidgets is an advanced jQuery, Angular, React, ASP .NET MVC, Custom Elements and HTML5 UI framework.",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/jqwidgets/jQWidgets"
+  },
+  "npmName": "jqwidgets-scripts",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "jqwidgets/*.*",
+        "jqwidgets/*/*.*",
+        "jqwidgets-ng/*.*",
+        "jqwidgets-ng/*/*.*",
+        "jqwidgets-ng/*/*/*.*",
+        "jqwidgets-ng/*/*/*/*.*",
+        "jqwidgets-react/*.*",
+        "jqwidgets-react-tsx/*/*.*",
+        "jqwidgets-ts/*.*",
+        "jqwidgets-vue/*.*"
+      ]
+    }
+  ],
+  "keywords": [
+    "JavaScript",
+    "jQuery",
+    "HTML5",
+    "jQWidgets",
+    "UI",
+    "widget",
+    "Angular",
+    "Grid",
+    "enterprise",
+    "table",
+    "data",
+    "PivotGrid",
+    "Chart",
+    "Scheduler",
+    "angular grid",
+    "react grid",
+    "React",
+    "ReactJS",
+    "angular5",
+    "angular4",
+    "angular6",
+    "angular 7",
+    "angular7",
+    "angular 7 grid",
+    "angular datagrid",
+    "angular chart",
+    "angular scheduler",
+    "angular calendar",
+    "angular cli",
+    "react tsx",
+    "web components",
+    "custom elements",
+    "html elements",
+    "react datagrid",
+    "react typescript",
+    "typescript",
+    "User Interface",
+    "UI Components",
+    "UI Widgets",
+    "jquery grid",
+    "javascript grid",
+    "javascript chart",
+    "jquery chart",
+    "jquery pivotgrid",
+    "AngularJS",
+    "Vue.js",
+    "Vue",
+    "TreeGrid",
+    "DataTable",
+    "TreeMap",
+    "Editor",
+    "Ribbon",
+    "Layout",
+    "DockingLayout",
+    "Tabs",
+    "Menu",
+    "Tree",
+    "NavigationBar",
+    "ListMenu",
+    "ToolBar",
+    "ComboBox",
+    "DropDownList",
+    "ListBox",
+    "Buttons",
+    "TextArea",
+    "Input",
+    "PasswordInput",
+    "MaskedInput",
+    "ComplexInput",
+    "FormattedInput",
+    "NumberInput",
+    "DateTimeInput",
+    "Calendar",
+    "Kanban",
+    "Window",
+    "Docking",
+    "Notification",
+    "PopOver",
+    "Tooltip",
+    "ColorPicker",
+    "ScrollView",
+    "ProgressBar",
+    "FileUpload",
+    "BulletChart",
+    "Knob",
+    "Gauge",
+    "BarGauge",
+    "Expander",
+    "Rating",
+    "RangeSelector",
+    "Slider",
+    "ScrollBar",
+    "Splitter",
+    "Sortable",
+    "ResponsivePanel",
+    "NavBar",
+    "TagCloud",
+    "Loader",
+    "Draw",
+    "DragDrop",
+    "DataAdapter",
+    "Response",
+    "Validator",
+    "Panel"
+  ],
+  "author": {
+    "name": "jQWidgets",
+    "email": "support@jqwidgets.com",
+    "url": "http://www.jqwidgets.com/"
+  },
+  "license": "http://www.jqwidgets.com/license/",
+  "homepage": "http://www.jqwidgets.com/",
+  "filename": ""
```

Comment:
Is there no default file that you can recommend users should use?

Reply:
Your feedback has been implemented.